scripts/gm/bin/lib-gm

```diff
@@ -995,8 +995,8 @@ rpc_timeout = '15s'
 account_prefix = '${ACCOUNT_PREFIX}'
 key_name = 'wallet'
 store_prefix = 'ibc'
-gas_price = { price = 0.001, denom = '${DENOM}' }
-max_gas = 1000000
+gas_price = { price = 0.01, denom = '${DENOM}' }
+max_gas = 10000000
```

Comment:
Note to reviewers: this change is unrelated to bug #2262.

It's just a parameter change so Hermes can submit bigger transactions by default (allows bigger gas usage).